napari/layers/_vispy/vispy_image_layer.py

```diff
@@ -0,0 +1,35 @@
+from vispy.scene.visuals import Image as ImageNode
```

Comment:
The `_vispy` folder should not be in `layers`, as it is a view. Only models should be in `layers`, at least as I understand the purpose of this refactor...?

Reply:
I know you've acknowledged this comment, but you haven't really discussed it or done anything about it... =)

Reply:
I've moved this back to the top level - I don't have strong feelings about it. We've already pulled qt layer files from the layers folder into the `_qt` folder so it's fine to have these separate too. We could have a separate folder called `_views` that contains both `_qt` and `_vispy`, but maybe that's making things too nested. Some of this stuff I think we can revisit when we start improving our docs and thinking about where everything lives and is grouped there